pkg/storage/snapshot/snapshot.go

```diff
@@ -898,7 +907,27 @@ func (ctrl *VMSnapshotController) updateVolumeSnapshotStatuses(vm *kubevirtv1.Vi
 	log.Log.V(3).Infof("Update volume snapshot status for VM [%s/%s]", vm.Namespace, vm.Name)
 
 	vmCopy := vm.DeepCopy()
-	volumes, err := storageutils.GetVolumes(vmCopy, ctrl.Client, storageutils.WithAllVolumes)
+
+	opts := []storageutils.VolumeOption{storageutils.WithRegularVolumes}
+	exists, err := ctrl.checkVMIRunning(vm)
```

Comment:
I dont think for the volumesnapshotstatus you need to distinguish between online/offline.. this status represent the option to snapshot a volume which can be snapshoted when offline

Reply:
Do you mean we should include the backend PVC here also if the VM is online?

Reply:
yes